test/liblll/Parser.cpp

```diff
@@ -180,6 +181,290 @@ BOOST_AUTO_TEST_CASE(macro_with_zero_args)
+    "600060ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff9a50505050505050505050505060006000f3",
+    "600060ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff9b5050505050505050505050505060006000f3",
+    "600060ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff9c505050505050505050505050505060006000f3",
+    "600060ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff9d50505050505050505050505050505060006000f3",
+    "600060ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff9e5050505050505050505050505050505060006000f3",
+    "600060ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff60ff9f505050505050505050505050505050505060006000f3",
+    "60006000a060006000f3",
+    "60ff60006000a160006000f3",
+    "60ff60ff60006000a260006000f3",
+    "60ff60ff60ff60006000a360006000f3",
+    "60ff60ff60ff60ff60006000a460006000f3",
+    "6000600060fff05060006000f3",
+    "60006000600060006017730f572e5295c57f15886f9b263e2f6d2d6c7b5ec66064f15060006000f3",
+    "60006000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec66064f25060006000f3",
+    "60006000f3",
+    "6000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec6620186a0f45060006000f3",
+    "6000600060006000730f572e5295c57f15886f9b263e2f6d2d6c7b5ec6612710fa5060006000f3",
+    "60006000fd60006000f3",
+    "32ff"
+  };
+
+  std::vector<std::string> opcodes_lll {
+    "{ (STOP) (RETURN 0x0 0x1) }",
+    "{ (ADD 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (MUL 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (SUB 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (DIV 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (SDIV 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (MOD 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (SMOD 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (ADDMOD 0x01 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (MULMOD 0x01 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (EXP 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (SIGNEXTEND 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (LT 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (GT 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (SLT 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (SGT 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (EQ 0x01 0x01) (RETURN 0x0 0x0) }",
+    "{ (ISZERO 0x00) (RETURN 0x0 0x0) }",
+    "{ (AND 0x00 0x00) (RETURN 0x0 0x0) }",
+    "{ (OR 0x00 0x00) (RETURN 0x0 0x0) }",
+    "{ (XOR 0x00 0x00) (RETURN 0x0 0x0) }",
+    "{ (NOT 0x00) (RETURN 0x0 0x0) }",
+    "{ (BYTE 0 0x8050201008040201) (RETURN 0x0 0x0) }",
+    //"{ (SHA3 0x00 0x00) (RETURN 0x0 0x0) }", //SHA3 opcode not implemented
+    "{ (ADDRESS) (RETURN 0x0 0x0) }",
+    "{ (BALANCE 0x0) (RETURN 0x0 0x0) }",
+    "{ (ORIGIN) (RETURN 0x0 0x0) }",
+    "{ (CALLER) (RETURN 0x0 0x0) }",
+    "{ (CALLVALUE) (RETURN 0x0 0x0) }",
+    "{ (CALLDATALOAD 0x0) (RETURN 0x0 0x0) }",
+    "{ (CALLDATASIZE) (RETURN 0x0 0x0) }",
+    "{ (CALLDATACOPY 0x0 0x0 0x0) (RETURN 0x0 0x0) }",
+    "{ (CODESIZE) (RETURN 0x0 0x0) }",
+    "{ (CODECOPY 0x1 0x0 (CODESIZE)) [[ 0 ]] (MLOAD 0x1) (RETURN 0x0 0x0) }",
```

Comment:
Why mix the built in notation of `sstore` with the rest? This code right now reads as
```
(CODECOPY 0x1 0x0 (CODESIZE))
(SSTORE 0 (MLOAD 0x1))
(RETURN 0x0 0x0)
```

is that intended?